apis/kueue/v1beta1/fairsharing_types.go

```diff
@@ -49,4 +53,12 @@ type FairSharingStatus struct {
 	// weight of zero, this will return 9223372036854775807, the
 	// maximum possible share value.
 	WeightedShare int64 `json:"weightedShare"`
+
```

Comment:
Same as with config, what about putting this in context of admission-usage-based-fs?